lib/rules/font-weight-notation/index.js

```diff
@@ -133,6 +138,17 @@ const rule = function(expectation, options) {
   };
 };
 
+function replaceWord(optionName, input) {
+  if (optionName === "named-where-possible") {
+    return input.replace(/400/g, NORMAL_KEYWORD).replace(/700/g, BOLD_KEYWORD);
+  }
+  if (optionName === "numeric") {
+    return input
+      .replace(/normal/g, WEIGHTS_WITH_KEYWORD_EQUIVALENTS[0])
+      .replace(/bold/g, WEIGHTS_WITH_KEYWORD_EQUIVALENTS[1]);
```

Comment:
This line is not good... I'll fix.(e.g. `font: var(--bold-x) italic 20px;`)

Reply:
@abouthiroppy Can you add tests and fix this?

Reply:
Sure. BTW, Should we also support auto fix other than CSS?

Reply:
@abouthiroppy where only standard syntax